heed/src/env.rs

```diff
@@ -280,28 +332,36 @@ impl Env {
                 .map(|db| PolyDatabase::new(self.env_mut_ptr() as _, db)))
     }
 
-    fn raw_open_database(
+    fn raw_open_dbi(
```

Comment:
Would it be possible to use this function in `raw_create_database` too? And also in all other places where this is useful?